stress-example/image/query_annoy_binarypb.yml

```diff
@@ -0,0 +1,34 @@
+!Flow
+version: '1'
+with:
+  read_only: true  # better add this in the query time
+  rest_api: false
+  port_expose: $JINA_PORT
+pods:
+  - name: encoder
+    uses: encoder.yml
+    shards: $JINA_SHARDS_ENCODER
+    host: $JINA_ENCODER_HOST
+    port_expose: 8000
+    timeout_ready: 600000
+  - name: vec_idx
+    uses: docker://jinahub/pod.indexer.annoyindexer:0.0.16-0.9.29
```

Comment:
Since we're using this all over, maybe we should make it an env var. `ANNOY_IMAGE`?

Reply:
done, can u review again?